engine-extensions/src/main/java/io/nosqlbench/engine/extensions/globalvars/GlobalVarsWrapper.java

```diff
@@ -0,0 +1,20 @@
+package io.nosqlbench.engine.extensions.globalvars;
+
+import io.nosqlbench.virtdata.library.basics.core.threadstate.SharedState;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+public class GlobalVarsWrapper {
+    ConcurrentHashMap<String, Object> map = SharedState.gl_ObjectMap;
+
+    String test = "puppies";
```

Comment:
too many puppies

Reply:
Whoops didn't mean to commit this class, removing.